src/si/volume.rs

```diff
@@ -105,6 +105,19 @@ quantity! {
 
 #[cfg(test)]
 mod tests {
+    #[test]
+    fn test_volumn_units() {
```

Comment:
```suggestion
    fn test_volume_units() {
```

Because this test is really for the `Units` enum and `units()` method move to `src/tests/quantity.rs`. I also suggest renaming to something like `test_units_enum` or `test_units_iterator`.